govcd/media.go

```diff
@@ -637,3 +637,42 @@ func (mediaRecord *MediaRecord) Delete() (Task, error) {
 	return mediaRecord.client.ExecuteTaskRequest(mediaRecord.MediaRecord.HREF, http.MethodDelete,
 		"", "error deleting Media item: %s", nil)
 }
+
+// QueryAllMedia returns media image found in system using `name` as query.
+func (vdc *Vdc) QueryAllMedia(mediaName string) ([]*MediaRecord, error) {
+	util.Logger.Printf("[TRACE] Querying medias by name\n")
+
+	if mediaName == "" {
+		return nil, errors.New("media name is empty")
+	}
+
+	typeMedia := "media"
```

Comment:
Just curious - can there be a mix of media (admin and non-admin) when a user is admin?

Reply:
I think you query simple and type admin type as sys admin

Reply:
My point would be that name `QueryAllMedia` is misleading if an admin can query both, but would only get results from `adminMedia`. Although I don't know if this is a true statmenet

Reply:
My name intention was AllMedia - not bound by catalog.